### PR TITLE
Migrate to babel config and updgrade webpack-vue-config

### PR DESCRIPTION
**Pull request by skjnldsv**
Update global configs :warning:

### CODE REVIEW

**Comment by skjnldsv** [comment]
@juliushaertl some deps needs fixing too

```bash
$ npm audit fix
npm WARN @nextcloud/vue-dashboard@1.1.0 requires a peer of @nextcloud/vue@^2.3.0 but none is installed. You must install peer dependencies yourself.

fixed 3818 of 3818 vulnerabilities in 2603 scanned packages
```

File: webpack.js [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
+webpackConfig.entry = {
+	...webpackConfig.entry,
+	collections: path.join(__dirname, 'src', 'init-collections.js'),
+	dashboard: path.join(__dirname, 'src', 'init-dashboard.js'),
+	calendar: path.join(__dirname, 'src', 'init-calendar.js'),
+	talk: path.join(__dirname, 'src', 'init-talk.js'),
 }
 
-module.exports = merge(webpackConfig, config)
+webpackConfig.resolve.extensions.push('.json')
```
Comment (skjnldsv) [inline webpack.js:12]: Do you actually import json files?

Comment (juliushaertl) [reply webpack.js:12]: Not that i remember, but let me drop it since removing doesn't have any effect on the resulting bundle :grin:

**Review by juliushaertl — APPROVED** [review]
Thanks a lot :+1:

**Pull request merged**